dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
```

Comment:
Do you need to explicitly block the label property from the base.yaml?

Existing [example](https://github.com/zephyrproject-rtos/zephyr/blob/81908cd36747da5a3c4c649e79decf4facbd6a4e/dts/bindings/test/vnd%2Cgpio-device.yaml#L11)

Reply:
Hmmm, I don't know. I didn't see any build warnings or errors without it. @galak?

Reply:
Yes, you are right. I didn't see the build warning until I actually tried to set the label property on a sensor node in the devicetree. I reverted back to `friendly-name` and added a discussion about why in the commit message:

```
An earlier version of this patch attempted to use the label property
instead of friendly-name, as it was noted during code review as a
possibly legitimate usage of the mostly-deprecated label property.
However, in practice it was difficult to implement because most sensor
bindings also inherit from i2c-device.yaml or spi-device.yaml, and
therefore inherit the deprecated label property from base.yaml. To work
around the deprecation, every sensor binding would have needed to
explicitly block the label property with a property-blocklist, which
would somewhat defeat the purpose of having a shared sensor-device.yaml.
```